Tools/CISupport/ews-build/steps.py

```diff
@@ -1722,6 +1722,48 @@ def hideStepIf(self, results, step):
         return not self.doStepIf(step)
 
 
+class RemoveLabelsFromPullRequest(buildstep.BuildStep, GitHubMixin):
+    name = 'remove-labels-from-pull-request'
+    flunkOnFailure = False
+    haltOnFailure = False
+
+    @defer.inlineCallbacks
+    def _addToLog(self, logName, message):
+        try:
+            log = self.getLog(logName)
+        except KeyError:
+            log = yield self.addLog(logName)
+        log.addStdout(message)
+
+    def start(self):
+        pr_number = self.getProperty('github.number', '')
+        build_finish_summary = self.getProperty('build_finish_summary', None)
+
+        rc = SKIPPED
+        if CURRENT_HOSTNAME == EWS_BUILD_HOSTNAME:
+            repository_url = self.getProperty('repository', '')
+            rc = SUCCESS
+            if not self.remove_labels(pr_number, [self.MERGE_QUEUE_LABEL, self.FAST_MERGE_QUEUE_LABEL, self.BLOCKED_LABEL], repository_url=repository_url):
```

Comment:
can explicitly store the list of labels to remove as a class variable, so it's very clear and easy to modify in future